src/RieSrf/RiemannSurface.jl

```diff
@@ -1119,17 +1139,17 @@ function symplectic_reduction(K::ZZMatrix)
       v = -A[pivot, j]
       if v != 0
         #The version with ! gave different results for some reason.
-        add_row!(A, v, pivot_plus, j)
-        add_column!(A, v, pivot_plus, j)
-        add_row!(B, v, pivot_plus, j)
+        A = add_row(A, v, pivot_plus, j)
+        A = add_column(A, v, pivot_plus, j)
+        B = add_row(B, v, pivot_plus, j)
```

Comment:
Could you please investigate? If there is something fishy we should know what is wrong.

Reply:
I cannot reproduce this behavior. Maybe you had an outdated Nemo version?
In some recent version the indices were inconsistent with the doc and abstract algebra 
We had that fixed.